test/widgets/compose_box_test.dart

```diff
@@ -75,6 +62,25 @@ void main() {
     controller = tester.state<ComposeBoxState>(find.byType(ComposeBox)).controller;
   }
 
+  Future<void> prepare(WidgetTester tester, {
```

Comment:
```
compose test [nfc]: Extract prepareComposeBox
```

In a file that's entirely about compose-box tests (compose_box_test.dart), and in the same scope (just at the top of the `main` method), it feels confusing to have one function named `prepare` and another named `prepareComposeBox`. If I were writing a test, I don't think I would know which function to use, except by reading their implementations.

How about instead of this refactor, we make `prepare` more flexible (if needed) so it can be used in the new tests?